tools/integration_tests/run_tests_mounted_directory.sh

```diff
@@ -137,3 +137,13 @@ sudo umount $MOUNT_DIR
 gcsfuse --only-dir testDir --implicit-dirs $TEST_BUCKET_NAME $MOUNT_DIR
 GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/list_large_dir/...  -p 1 --integrationTest -v --mountedDirectory=$MOUNT_DIR --testbucket=$TEST_BUCKET_NAME/testDir
 sudo umount $MOUNT_DIR
+
+# Run integration tests for read_large_files directory with static mounting
+gcsfuse --implicit-dirs $TEST_BUCKET_NAME $MOUNT_DIR
```

Comment:
Also, you are running for more sets of flags, so commands for those should also be added here.

Reply:
Added.